src/pages/profile.tsx

```diff
@@ -0,0 +1,66 @@
+import Head from "next/head";
+
+import { api } from "~/utils/api";
+
+export default function ProfilePage() {
+  const { data: user } = api.user.getMe.useQuery();
+
+  return (
+    <>
+      <Head>
+        <title>Profile</title>
+        <meta name="description" content="User Profile" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className=" flex  min-h-screen flex-col items-center justify-center bg-gradient-to-b from-[#2e026d] to-[#15162c] text-white">
+        {user ? <Profile {...user} /> : <p>loading...</p>}
+      </main>
+    </>
+  );
+}
+
+type ProfileProps = { name: string; id: number };
+function Profile(user: ProfileProps) {
+  return (
+    <div className="container flex flex-col items-center  justify-center  gap-12 px-4 py-16 text-white ">
+      <h1 className="text-5xl font-extrabold tracking-tight sm:text-[5rem]">
```

Comment:
In case we want to build a more accessible solution, would there be an HTML tag more suitable for this?